controller/plugins/controller/kubejobs/plugin.py

```diff
@@ -88,3 +90,31 @@ def stop_application_scaling(self):
 
     def status(self):
         return self.alarm.status()
+
+    def validate(self, data):
```

Comment:
Shouldn't we add this method in `controller.plugins.controller.base` (i.e. should all controllers validate it's info?)
If so, this could be parametrized.

Reply:
No, because the plugin structure isn't consistent, and since we agreed that we aren't going to do structural changes, this was the best solution.